libraries/ms-common/test/test_can.c

```diff
@@ -275,3 +278,49 @@ void test_can_default(void) {
   TEST_ASSERT_EQUAL(msg.msg_id, rx_msg.msg_id);
   TEST_ASSERT_EQUAL(msg.data, rx_msg.data);
 }
+
+#ifdef X86
+void test_can_x86_tx(void) {
+  volatile CanMessage rx_msg = { 0 };
+  CanMessage msg = {
+    .msg_id = 0xF,              //
+    .type = CAN_MSG_TYPE_DATA,  //
+    .data = 0x1,                //
+    .dlc = 1,                   //
+  };
+
+  can_register_rx_handler(0xF, prv_rx_callback, &rx_msg);
+  s_rx_cb_count = 0;
```

Comment:
Resetting `s_rx_cb_count` should be done in `setup_test`, even if it's only used here.